x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
+    border-right: $euiBorderThin;
+  }
+
+  &__contentInner {
+    display: grid;
+    grid-gap: $euiSizeS;
+    overflow: hidden;
+  }
+
+  &__fieldsetContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__hiddenFieldsIndicator {
+    font-size: $euiFontSizeXS;
+    color: $euiColorDarkShade;
+    align-self: end;
+    padding-top: 0;
+  }
+
+  &__toggleExpandButton {
+    justify-self: stretch;
+    align-self: stretch;
+    display: flex;
+    width: $euiSizeL * 2;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: $euiPageBackgroundColor;
+      cursor: pointer;
+    }
+  }
+
+  &__toggleExpandButton {
+    grid-column: 2;
+  }
+
```

Comment:
On closer inspection this actually isn't being used, the parent wrapper isn't a grid, it's flex - we can now safely remove it 🎉 

```suggestion
```